src/source/patch.rs

```diff
@@ -622,6 +559,12 @@ mod tests {
         #[exclude("mumps")]
         // Failed to download source
         #[exclude("petsc")]
+        // GNU patch fails and diffy succeeds, seemingly correctly from the diff output.
+        #[exclude("(fastjet-cxx)|(fenics-)|(love2d)|(flask-security-too)")]
```

Comment:
Would be nice to at least test `love2d` as it was the one we had trouble with for `CLRF`.

We don't have to run it against Gnu Patch - just add some kinda snapshot test or assert that the CMakeLists file changed in the way we expect.